### PR TITLE
Plot3

### PR DESCRIPTION
**Pull request by Farnazmdi**
Plot for phenotypes. It still needs some modifying for identifying the shape markers, but I guess it should be ready to merge for now.

### CODE REVIEW

**Comment by Farnazmdi** [comment]
image

**Comment by aarmey** [comment]
Have @shak360 review.

File: lineage/Analyze.py [hunk 1]
```diff
@@ -193,7 +193,9 @@ def run_Results_over(output):
     :type output: list
     """
     results_holder = []
+    pred_states_bylin = []
     for _, (tHMMobj, pred_states_by_lineage, LL) in enumerate(output):
         results_holder.append(Results(tHMMobj, pred_states_by_lineage, LL))
+        pred_states_bylin.append(pred_states_by_lineage)
 
-    return results_holder
+    return results_holder, pred_states_bylin
```
Comment (shak360) [inline lineage/Analyze.py:201]: Ideally, this should be stored in the `results_holder` but I can add that on a new branch.

Comment (shak360) [reply lineage/Analyze.py:201]: This would avoid having the `nothing` argument in the other functions.

Comment (Farnazmdi) [reply lineage/Analyze.py:201]: Great, thanks!
I will merge this for now then.

**Pull request merged**